apps/api/src/app/notifications/activity.controller.ts

```diff
@@ -38,18 +38,18 @@ export class ActivityController {
     @UserSession() user: IJwtPayload,
     @Query() query: ActivitiesRequestDto
   ): Promise<ActivitiesResponseDto> {
-    let channelsQuery: ChannelTypeEnum[];
+    let channelsQuery: ChannelTypeEnum[] | undefined;
```

Comment:
```suggestion
    let channelsQuery;
```

Reply:
it will be any which is not appropriate type